association_backend/association/stripe/tests/fixtures/checkout_session__completed.py

```diff
@@ -0,0 +1,57 @@
+test_data = {
```

Comment:
I am a bit conflicted about this, I need to read the tests a bit more

Reply:
this was a typo, these fixtures are not used
I removed them